API.md

```diff
@@ -398,11 +406,12 @@ mode |  simple_select, direct_select, draw_point, draw_line_string, draw_polygon
 
 Draw also provides a few more properties, but they should not be used for styling. For details on them, see `Using Draw with map.queryRenderFeatures`.
 
-### Example Custom Style
-
-With this style all Point features are blue and have a black halo when active. No other features are rendered, even if they are present.
+### Example Custom Styles
```

Comment:
More examples are great. Please move the comments out of the code blocks and make sure you are always wrapping the style JSON in `mapbox.Draw({...})` so that reader understand its javascript vs json.